api/v1alpha1/author_webhook_test.go

```diff
@@ -116,6 +116,36 @@ var _ = Describe("Author webhook", Ordered, func() {
 				Expect(patch.Path).To(ContainSubstring("author"))
 				Expect(patch.Value).To(Equal("admin"))
 			})
+
+			It("should not add the author label if the automated label is present and true", func() {
+				release.Labels = map[string]string{
+					metadata.AutomatedLabel: "true",
+				}
+				admissionRequest.Object.Raw, err = json.Marshal(release)
+				Expect(err).NotTo(HaveOccurred())
+
+				rsp := authWebhook.Handle(ctx, admissionRequest)
+				Expect(rsp.AdmissionResponse.Allowed).To(BeTrue())
+				Expect(rsp.AdmissionResponse.Patch).To(BeNil())
+				Expect(len(rsp.Patches)).To(Equal(0))
+			})
+
+			It("should add the author label if the automated label is false", func() {
+				release.Labels = map[string]string{
+					metadata.AutomatedLabel: "false",
+				}
+				admissionRequest.Object.Raw, err = json.Marshal(release)
+				Expect(err).NotTo(HaveOccurred())
+
+				rsp := authWebhook.Handle(ctx, admissionRequest)
+				Expect(rsp.AdmissionResponse.Allowed).To(BeTrue())
+				Expect(len(rsp.Patches)).To(Equal(1))
+				patch := rsp.Patches[0]
+				Expect(patch.Operation).To(Equal("add"))
+				// The json functions replace `/` so checking the entire value does not work
```

Comment:
I would remove this line. The same comment exists in the test above but not in the first test in the context. Maybe both are unnecessary.

Reply:
I put the comment to explain why we are comparing substrings instead of the entire string. The comment is not there when we are comparing the entire string because it wouldn't make sense in that case... the comment is about why are are using containsSubstring instead of Equals